src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -176,12 +176,15 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     peopleInput.focus(options);
     peopleInput.select();
 
-    // Mouse is focused on input
-    if (!peopleInput.value) {
-      this.hideFlyout();
-    } else {
-      this.showFlyout();
-    }
+    // The timeout is needed to force the layout update.
+    setTimeout(() => {
```

Comment:
You might also do this on the next render loop with requestAnimationFrame for a more optimized solution

Reply:
Good idea